packages/cli-lib/lang/en.lyaml

```diff
@@ -923,6 +937,13 @@ en:
           errors:
             invalidName: "You entered an invalid name. Please try again."
             projectDoesNotExist: "Project \"{{ projectName }}\" could not be found in \"{{ accountId }}\""
+        feedbackPrompt:
+          feedbackType:
+            message: "What type of feedback would you like to leave?"
+            bug: "[--bug] Report a bug"
+            general: "[--general] Tell us about your experience with projects"
```

Comment:
Now that this is more of a general feedback command, should we still explicitly mention that the feedback should be for projects here?

Reply:
Good question - this copy is from @markhazlewood so I'll double check